paper-menu.html

```diff
@@ -0,0 +1,330 @@
+<!--
+    @license
+    Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../core-selector/core-selectable.html">
+<link rel="import" href="../layout/x-layout.html">
+<link rel="import" href="../paper-styles/x-theme.html">
+
+<!--
+@element paper-menu
+-->
+
+<dom-module id="paper-menu">
+
+  <style>
+
+    :host {
+      padding: 8px 0;
+
+      background: var(--primary-background-color);
+      color: var(--primary-text-color);
+
+      mixin(--paper-menu);
+    }
+
+    :host > ::content > [disabled] {
+      color: var(--disabled-text-color);
+    }
+
+    :host > ::content > .core-selected {
+      position: relative;
+    }
+
+    :host > ::content > .core-selected::after {
+      mixin(--fit);
+
+      background: currentColor;
+      /* FIXME move to paper-styles for next widget */
+      opacity: 0.12;
+      content: '';
+
+      mixin(--paper-menu-selected-item);
+    }
+
+    :host > ::content > .core-selected[colored]::after {
+      opacity: 0.26;
+
+      mixin(--paper-menu-colored-selected-item);
+    }
+
+  </style>
+
+  <template>
+
+    <content></content>
+
+  </template>
+
+</dom-module>
+
+<script>
+
+(function() {
+
+  // FIXME menu control should be refactored to a more general element
+
+  Polymer({
+
+    is: 'paper-menu',
+
+    enableCustomStyleProperties: true,
```

Comment:
Does this belong in `properties`? If not, why not set it in `ready` or something similar?